packages/iov-ethereum/src/utils.ts

```diff
@@ -49,3 +49,11 @@ export function hexPadToEven(hex: string): string {
   }
   return hex.replace("0x", "");
 }
+
+export function trimLeadingZero(hex: string): string {
```

Comment:
Why does this function accept odd number of characters (e.g. "002") but does not trim odd number of leading zeros?

* if this is for bytes encoded as hex -> input length must be even
* if this is for integers encoded as hex -> all leading zeros should be trimmed